.eslintrc.json

```diff
@@ -37,23 +37,18 @@
         "leadingUnderscore": "allow"
       }
     ],
-    "no-tabs": "error",
+    "no-tabs": [
+      "error",
+      {
+        "allowIndentationTabs": true
+      }
+    ],
     "react/prop-types": "off",
     "@typescript-eslint/no-unused-vars": "error"
   },
   "overrides": [
     {
-      "files": [
-        "src/utils/shared/*.?(js|ts)",
-        "src/gatsby/**/*.js",
-        "scripts/**/*.js",
-        "config/**/*.js",
-        "src/server/**/*.js",
-        "plugins/**/*.js",
-        "gatsby-*.js",
-        "postcss.config.js",
-        "tailwind.config.js"
-      ],
+      "files": "**/*.js",
```

Comment:
This is how it is in other websites and makes sense as long as we have CJS mixed in.